internal/flypg/zombie.go

```diff
@@ -50,41 +52,87 @@ func readZombieLock() (string, error) {
 	return string(body), nil
 }
 
-func ZombieDiagnosis(myHostname string, total int, inactive int, active int, conflictMap map[string]int) (string, error) {
+type DNASample struct {
+	hostname       string
+	totalMembers   int
+	totalActive    int
+	totalInactive  int
+	totalConflicts int
+	conflictMap    map[string]int
+}
+
+func TakeDNASample(ctx context.Context, node *Node, standbys []Member) (*DNASample, error) {
```

Comment:
nice :)

Reply:
really leaning into the zombie theme. :D